### DISCUSSION TITLE
4 technologies we may or should use in the future

### DISCUSSION THREAD

**Discussion by francoiscrouzet** [archaiodata/thacer#16] [Ideas]
- pinia https://pinia.vuejs.org/
- composition api https://vuejs.org/guide/extras/composition-api-faq.html#what-is-composition-api
- nuxt https://nuxtjs.org/
- typescript https://www.typescriptlang.org/
- vueuse https://vueuse.org/

**Comment by AlZorba** [comment]
in deed nice thread to have for listing potential helpful technologies. Maybe we can keep it simple but add a small description or a link?

**Reply by francoiscrouzet** [reply]
Done :)
